src/madpack/create_changelist.py

```diff
@@ -237,6 +325,13 @@
     print "Something went wrong! The changelist might be wrong/corrupted."
     raise
 finally:
-    os.system("rm -f /tmp/madlib_tmp_nm.txt /tmp/madlib_tmp_udf.txt "
-              "/tmp/madlib_tmp_udt.txt /tmp/madlib_tmp_cl.yaml "
```

Comment:
what is this madlib_tmp_cl.yaml file here? why we don't remove it now

Reply:
Nice catch, it should still be removed.